CHANGES.rst

```diff
@@ -22,6 +22,7 @@ Features
 Fixes
 -----
 
+- Make example run from any directory (#377)
```

Comment:
I ended up moving this to sit with the other recent changes in documentation.  I'm mentally classifying examples as documentation and figured these three related issues should all sit together